SQLQuery_1.sql

```diff
@@ -0,0 +1,49 @@
+
+CREATE DATABASE FindYourRecipe;
+CREATE TABLE Recipes 
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Title VARCHAR(50),
+    Instructions VARCHAR(MAX),
+    Category VARCHAR(30),
+    Dificulty VARCHAR(30)
+);
+
+CREATE TABLE Ingredients 
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Name VARCHAR(50),
+    Category VARCHAR(30)
```

Comment:
Same with this `Category`. There should be a limited set of values allowed (example: `meat`, `diaries`. etc) and by having this info in another table we enforce data integrity which means that we won't allow the user to create an ingredient of category `couch` which would make absolutely no sense.